content/docs/integrations/aws.txt

```diff
@@ -1,122 +0,0 @@
----
```

Comment:
I am unsure of what this file is. It looks like it might have been an initial draft of the aws elbs integration but it doesnt seem fleshed out. I removed it while I was working to ignore it but want to make sure isnt a file we need before I actually delete it?